<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
background-image: url(://fondo.png);
}

#instrucciones {
color: white;
}

#cronometro {
color: white;
}

#puntuacion {
color: white;
}

#lblPuntaje {
color: white;
qproperty-alignment: AlignCenter;
background: teal;
border: 3px solid silver;
border-radius: 7px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="instrucciones">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>70</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Click en dos tarjetas para intentar que coincidan</string>
    </property>
   </widget>
   <widget class="QLabel" name="cronometro">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>cron√≥metro</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>129</y>
      <width>631</width>
      <height>401</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="tarjeta01">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta02">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta04">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta03">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta06">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta07">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta05">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta08">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>130</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta11">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta09">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta12">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>260</y>
       <width>130</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="puntuacion">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>518</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Puntos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPuntaje">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>515</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
